VstsClientLibrariesSamples/IConfiguration.cs

```diff
@@ -1,15 +1,18 @@
-﻿using System;
+ sing System;
```

Comment:
This is a compilation error. Do we have CI set up on this repo? If not, we should set up VSTS build. :)